pennylane/devices/capabilities.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
```

Comment:
Is there a plan to remove the now unnecessary functions in Catalyst and use the one from here instead?

Reply:
Yes, that is the idea. Once this and https://github.com/PennyLaneAI/pennylane/pull/6433 are merged into PL, I will open a PR in Catalyst that replaces all uses of the current TOML module with an import from PL.

Reply:
Doesn't look like it at this point. I will ask for support if I am blocked.